HelperClasses/RequestParameters.cs

```diff
@@ -1,60 +1,68 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
 
 namespace EncompassREST.HelperClasses
 {
     public class RequestParameters : List<RequestParameter>
     {
-
         public RequestParameters()
         {
-
         }
 
-        public void Add(string Key, string Value)
+        public void Add(string key, string value)
         {
-            this.Add(new RequestParameter(Key, Value));
+            Add(new RequestParameter(key, value));
         }
 
         public override string ToString()
         {
-            if (this.Count > 0)
+            if (Count > 0)
             {
-                StringBuilder sb = new StringBuilder();
-                sb.Append("?");
-                sb.Append(String.Join("&",this));
+                var sb = new StringBuilder();
```

Comment:
ok. check it in, i will double check this when I am in the office tomorrow, and merge it.